client/my-sites/site-settings/form-writing.jsx

```diff
@@ -140,20 +136,6 @@ const SiteSettingsFormWriting = React.createClass( {
 				</SectionHeader>
 				<Card className="site-settings">
 					<FormFieldset>
-						<QueryTerms siteId={ this.props.siteId } taxonomy="category" />
```

Comment:
Ha!  See I just didn't remember that - if that is the case, let's let it be and we can update the docs accordingly 👍